src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/JwtFilter.java

```diff
@@ -34,7 +34,7 @@ protected void doFilterInternal(
       final HttpServletResponse response,
       final FilterChain filterChain) throws ServletException, IOException {
 
-    String authorizingToken = request.getHeader("Authorization");
+    String authorizingToken = request.getHeader("authorization");
```

Comment:
Why did you make this lowercase? I think uppercase is the standard, no?

Reply:
nope lemmy uses lowercase ;)